scipy/stats/stats.py

```diff
@@ -2034,9 +2039,9 @@ def percentileofscore(a, score, kind='rank'):
     Parameters
     ----------
     a : array_like
-        Array of scores to which `score` is compared.
-    score : int or float
-        Score that is compared to the elements in `a`.
+        Array to which `score` is compared.
+    score : array_like
+        Array to compute percentiles for.
```

Comment:
```suggestion
        Scores to compute percentiles for.
```
I think this emphasizes that this is the 'score' referred to in the name 'percentileofscore'